src/components/App.tsx

```diff
@@ -1,36 +1,25 @@
+import { Entity } from '@backstage/catalog-model';
 import React from 'react';
-import { Route, MemoryRouter, Routes } from 'react-router';
-import { BuildsPage, Builds } from '../pages/BuildsPage';
-import { AppStateProvider } from '../state';
-import { Settings } from './Settings';
+import { Builds } from '../components/BuildsPage';
+import { ContextProvider } from './ContextProvider';
+import { LastBuildCard } from './LastBuildCard';
 
-export const App = () => {
+type Props = {
+  entity: Entity;
+};
+
+export const App: React.FC<Props> = ({ entity }) => {
```

Comment:
We should rename this file & component to something more specific. TravisCIBuildsTable ? TravisCIBuildsPage